doc/Language/signatures.rakudoc

```diff
@@ -986,6 +986,59 @@ $sig1 = :($goal, $accuracy = $goal / 100);
 $sig2 = :(:$excludes = ['.', '..']);        # a new Array for every call
 =end code
 
+U<Was an argument passed for a parameter?>
+
+Table showing checks of whether an argument was passed for a given parameter:
+
+=begin table
+ Parameter kind | Example | Comment | Check for no arg passed
+ ============================================================
+ L<Slurpy|https://docs.raku.org/language/signatures#Slurpy_parameters>|`*@array`|Don't check using `.defined`|`if not @array`
+ Required|C<$foo>|Can't be omitted|(not applicable)
+ Optional|C<@bar = default>¹|Pick, and check, a suitable default¹|C<if @bar === default {...}>¹
```

Comment:
Instead of manually entering footnotes, please use the `N<>` syntax, and an appropriate footnote will be generated when rendering the page.

Reply:
```suggestion
 Optional|C<@bar = default>|Pick, and check, a suitable defaultN<A suitable default is an Object that has a distinct identity, as may be checked by the L<C<WHICH>|https://docs.raku.org/type/Mu#method_WHICH> method.>|C<if @bar === default {...}>
```